eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/configuration/batch/listeners/ListenersConfiguration.java

```diff
@@ -12,11 +13,19 @@
 
 @Configuration
 public class ListenersConfiguration {
+    @Value("${eva.count-stats.url}")
```

Comment:
I think we should be consistent and use the same approach we are using with the `InputParameters` instead of using `@Value` for getting this group of properties. I suggest creating a class named something like `CountParameters` and adding a configuration for that would look better in my opinion

Reply:
added a CountParameters class